### DISCUSSION TITLE
How can I get variables filled in bulk by multiple users via API?

### DISCUSSION THREAD

**Discussion by odaviribeiro1** [langgenius/dify#3475] [General]
I've created an application in which the user can only interact with the chatbot if they first provide their name, email address and telephone number. Now I need to retrieve this information provided by users via API and store it in a database. I haven't found an efficient way of doing this in the documentation.

**Comment by guchenhe** [comment]
This sounds achievable with chatflow - you might be able an HTTP request method to write the user variables into a database. have you tried this?

**Reply by odaviribeiro1** [reply]
I dont tried this. Can you send me the link of chatflow?

**Comment by YidaHu** [comment]
@odaviribeiro1 
![image](https://github.com/langgenius/dify/assets/13404367/d8b077de-42e5-4172-9825-e7ab28274429)
![image](https://github.com/langgenius/dify/assets/13404367/24ed1d13-1018-4061-9a8c-f0e9f5e08656)
![image](https://github.com/langgenius/dify/assets/13404367/8bece0ce-9abb-4d2a-97d7-65d0ad291dd3)
